libs/nillion-chain/client/src/client.rs

```diff
@@ -438,3 +440,33 @@ impl NillionChainClient {
         client.abci_query(path, data, None, false).await.map_err(Into::into)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use rstest::rstest;
+    use crate::client::NillionChainClient;
+
+    #[rstest]
+    #[case(4000, 0.025, 0, 100, "No rounding up for exact multiple")]
```

Comment:
I meant using something like `case::exact_multiple`, not having an actual string. You can keep it this way, just throwing it out there.

Reply:
ah right, i forgot but this neat macro!